### DISCUSSION TITLE
Dependency Track receiving a 400 Bad Request from Okta with OpenID Configuration

### DISCUSSION THREAD

**Discussion by naga5645** [DependencyTrack/dependency-track#1282] [Q&A]
Hi Team,

We are trying to setup the Okta authentication with OpenID for Dependency track 4.3.6, after all the config changes are made and when we hit the Dependency track url we are getting 400 bad request from Okta.
Error Code: invalid_request
Description: The 'redirect_uri' parameter must be a Login redirect URI in the client app settings: https://admin.okta.com/admin/app/oidc_client/instance/0oa1mwpkgvowHgb2P0h8#tab-general

and our redirect and  login URI is : https://dependencytrack-uat.internal.io/static/oidc-callback.html

![image](https://user-images.githubusercontent.com/78976261/143269782-2b415e48-9da4-465c-9ee0-01de409ec4b4.png)
Could you please help us with this issue and its a blocker for us now.
Thanks,
Naga.

**Comment by arudinskis** [comment]
Hi,

Any updates regarding this issue? I'm trying to implement Okta OIDC but it doesn't work if Dependency Track deployed using custom URLs. If deployed on `localhost` Okta OIDC works fine.
